OpenRA.Game/Graphics/CursorSequence.cs

```diff
@@ -34,12 +34,12 @@ public CursorSequence(FrameCache cache, string name, string cursorSrc, string pa
 			var cursorSprites = cache[cursorSrc];
 			Frames = cursorSprites.Skip(Start).ToArray();
 
-			if ((d.ContainsKey("Length") && d["Length"].Value == "*") || (d.ContainsKey("End") && d["End"].Value == "*"))
+			if ((d.TryGetValue("Length", out var yaml) && yaml.Value == "*") || (d.ContainsKey("End") && yaml.Value == "*"))
```

Comment:
```suggestion
			if ((d.TryGetValue("Length", out var length) && length.Value == "*") || (d.TryGetValue("End", out var end) && end.Value == "*"))
```

Reply:
`yaml` is used as a scratch variable for all the `d.TryGetValue("Something", ...` calls in this file. Reusing it saves me having to intoduce lots of extra variables and also avoids the naming conflicts later in e.g.
```c#
if (d.TryGetValue("X", out yaml)) // Changing this to 'out var x' would seem natural.
{
    Exts.TryParseIntegerInvariant(d["X"].Value, out var x); // but x is already declared here for the parsed value.
    ...
```

Reply:
In any case your change is wrong. It leaves in a `d.ContainsKey` and then checks the wrong `yaml` for being `'*'`.
I guess you want
```suggestion
			if ((d.TryGetValue("Length", out var yaml) && yaml.Value == "*") || (d.TryGetValue("End", out yaml) && yaml.Value == "*"))
```

Reply:
Fixed